src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/AttachCaseCallbackService.java

```diff
@@ -115,8 +115,10 @@ private void verifyExceptionRecordIsNotAttachedToCase(
             exceptionRecordJurisdiction
         );
 
-        if (isExceptionRecordAttachedToCase(fetchedExceptionRecord)) {
-            throw new AlreadyAttachedToCaseException("Exception record is already attached to a case");
+        Object attachToCaseRef = fetchedExceptionRecord.getData().get(ATTACH_CASE_REFERENCE_FIELD_NAME);
```

Comment:
should we typecast this to string ?

Reply:
I guess. Just moved the existing code from lines 158 - 164 so that the change is minimal